src/mapml-viewer.js

```diff
@@ -354,6 +354,18 @@ export class MapViewer extends HTMLElement {
       totalSize += 49;
       this._fullScreenControl = M.fullscreenButton().addTo(this._map);
     }
+    if (!this._geolocationButton){
+      this._geolocationButton = M.geolocationButton({
+        showPopup: false,
+        strings: {
+          title: "Locate"
```

Comment:
Don't forget to add localized strings to the mapml-extension.  "Locate" should be: "Show my location"